presto-docs/src/main/sphinx/release/release-306.rst

```diff
@@ -0,0 +1,76 @@
+===========
+Release 306
+===========
+
+General Changes
+---------------
+
+* Fix planning failure for queries containing a ``LIMIT`` after a global
+  aggregation. (:issue:`437`)
+* Fix missing column types in EXPLAIN output. (:issue:`328`)
+* Fix updating peak revocable memory reservation. (:issue:`413`)
```

Comment:
I don't understand what any of these `413` items mean

Reply:
@sopel39?

Reply:
`Fix updating peak revocable memory reservation`

Peak revocable memory was not updated previously at all.

`Fix blocking on memory when revocable memory is used.`

I think we can remove it. 

`Fix double memory accounting for spilling aggregations.`

For aggregates revocable memory was also accounted as user/system mem

`Fix excessive CPU usage that can occur when spilling for window functions.`

Spill could have been triggered by operator iself. In such case operator did not report being blocked thus there was active waiting via `getOutput`.